### PR TITLE
docs(graphql): update mercurius integration

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Proposals:
  - Added link for `graphql-mercurius` example.
  - Removed dependencie `fastify` for mercurius installation, since you just need to install `@nestjs/platform-fastify`, for more information see `package.json` of this [example](https://github.com/nestjs/nest/blob/master/sample/33-graphql-mercurius/package.json)

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: content/graphql/quick-start.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ $ npm i @nestjs/graphql @nestjs/apollo graphql apollo-server-express
 # npm i @nestjs/graphql @nestjs/apollo graphql apollo-server-fastify
 
 # For Fastify and Mercurius
-# npm i @nestjs/graphql @nestjs/mercurius graphql mercurius fastify
+# npm i @nestjs/graphql @nestjs/mercurius graphql mercurius
```
Comment (kamilmysliwiec) [inline content/graphql/quick-start.md:19]: Why did you remove fastify?

File: content/graphql/quick-start.md [hunk 2]
```diff
@@ -345,3 +345,7 @@ export class AppModule {}
 > info **Hint** Once the application is running, open your browser and navigate to `http://localhost:3000/graphiql`. You should see the [GraphQL IDE](https://github.com/graphql/graphiql).
 
 The `forRoot()` method takes an options object as an argument. These options are passed through to the underlying driver instance. Read more about available settings [here](https://github.com/mercurius-js/mercurius/blob/master/docs/api/options.md#plugin-options).
+
+#### Example
+
+A first example of fully working code with Mercurius is available [here](https://github.com/nestjs/nest/tree/master/sample/33-graphql-mercurius).
```
Comment (kamilmysliwiec) [inline content/graphql/quick-start.md:351]: Can we use the same sentence we used in other chapters?

File: content/graphql/quick-start.md (hunk 1 above)
Comment (Tony133) [reply content/graphql/quick-start.md:19]: It is not necessary to install it as a dependency, see guide official of Mercurius [here](https://mercurius.dev/#/docs/integrations/nestjs)


screenshot
